tools/autograd/templates/TraceType.cpp

```diff
@@ -0,0 +1,31 @@
+#include "torch/csrc/autograd/VariableTypeUtils.h"
+
+#include <ATen/TypeDefault.h>
+#include <torch/library.h>
+
+#include "torch/csrc/autograd/function.h"
+
+// ${generated_comment}
```

Comment:
Might be worth adding a (non-generated) comment that briefly describes what these kernels do?

Reply:
Added a link to `torch/csrc/jit/OVERVIEW.md`.